lib/views/search/widgets/search_result_widget.dart

```diff
@@ -59,7 +60,13 @@ class SearchResultWidget extends ConsumerWidget {
         onTap: () => ref.refresh(searchProvider),
         isLoading: searchResult.isLoading,
       ),
-      loading: () => const SearchResultShimmerWidget(),
+      loading: () {
+        if (searchQuery.search) {
```

Comment:
Should this be isLoading?

Reply:
This is to check if the user has started a search. The variable name has been changed to hasSearchStarted to avoid confusion.